emuu-client/src/components/CreatorsPage/Feeds.js

```diff
@@ -37,12 +37,79 @@ import Tab from "@material-ui/core/Tab";
 import TabContext from "@material-ui/lab/TabContext";
 import TabList from "@material-ui/lab/TabList";
 import TabPanel from "@material-ui/lab/TabPanel";
+import Pagination from "@mui/material/Pagination";
+import Stack from "@mui/material/Stack";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+
+const options = ["Recently Uploaded", "Most Viewed"];
+
+const ITEM_HEIGHT = 48;
+
+function LongMenu({ sort, setSort }) {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = (e) => {
+    setSort(e.target.innerText);
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
+      Sort By
+      <IconButton
+        aria-label="more"
+        id="long-button"
+        aria-controls={open ? "long-menu" : undefined}
+        aria-expanded={open ? "true" : undefined}
+        aria-haspopup="true"
+        onClick={handleClick}
+      >
+        <MoreVertIcon />
+      </IconButton>
+      <Menu
+        id="long-menu"
+        MenuListProps={{
+          "aria-labelledby": "long-button",
+        }}
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        PaperProps={{
+          style: {
+            maxHeight: ITEM_HEIGHT * 4.5,
+            width: "20ch",
+          },
+        }}
+      >
+        {options.map((option) => (
+          <MenuItem
+            key={option}
+            selected={option === sort}
+            onClick={handleClose}
+          >
+            {option}
+          </MenuItem>
+        ))}
+      </Menu>
+    </div>
+  );
+}
 
 function Feeds({ setVideo, setUserProfile }) {
   const [recentVideos, setRecentVideos] = useState([]);
+  const [topVideos, setTopVideos] = useState([]);
+  const [pages, setPages] = useState(undefined);
+  const [page, setPage] = useState(1);
```

Comment:
Pages and Page are too similar variables, try making it a little more descriptive to allow code to be read easier.